### PR TITLE
40 Split view

### PR DESCRIPTION
**Pull request by pontuslaestadius**

### CODE REVIEW

File: Application/src/view/DiagramView.java [hunk 1]
```diff
@@ -3,24 +3,36 @@
 import javafx.application.Platform;
 import javafx.collections.FXCollections;
 import javafx.collections.ObservableList;
+import javafx.scene.canvas.Canvas;
 import javafx.scene.control.Tab;
 import javafx.scene.control.TabPane;
+import javafx.scene.layout.BorderPane;
+import javafx.scene.layout.HBox;
+import jdk.nashorn.internal.runtime.regexp.joni.exception.ValueException;
```
Comment (ghost) [inline Application/src/view/DiagramView.java]: this line is preventing compilation

**Review by ghost — APPROVED** [review]

File: Application/src/model/Menu.java [hunk 2]
```diff
@@ -209,6 +216,32 @@ private void setEvents(Stage stage) {
...
+            if (button_deployment.getText().toLowerCase().contains("show")) {
+                button_deployment.setText("Hide deployment diagram");
+                dv.addDiagram(dv.DEPLOYMENT_DIAGRAM);
+            } else {
+                button_deployment.setText(text_deployment);
+                dv.removeDiagram(dv.DEPLOYMENT_DIAGRAM);
+            }
+
+        });
+
```
Comment (timjon) [inline Application/src/model/Menu.java:262]: Comments on line 219-244 plz

**Review by timjon — CHANGES_REQUESTED** [review]

File: Application/src/model/Menu.java (hunk 2 above)
Comment (pontuslaestadius) [reply Application/src/model/Menu.java:262]: Added

**Review by kosara-g — APPROVED** [review]
Don't forget to remove the test code in Draw when we've merged the class diagram class and the like.

**Comment by timjon** [comment]
Does not work properly with the tabs

**Review by timjon — APPROVED** [review]

**Pull request merged**